src/librustdoc/html/static/js/main.js

```diff
@@ -1746,7 +1841,15 @@ function preLoadCss(cssUrl) {
     // On larger, "desktop-sized" viewports (though that includes many
     // tablets), it's fixed-position, appears in the left side margin,
     // and it can be activated by resizing the sidebar into nothing.
-    const sidebarButton = document.getElementById("sidebar-button");
+    let sidebarButton = document.getElementById("sidebar-button");
+    const body = document.querySelector(".main-heading");
```

Comment:
```suggestion
    const body = document.getElementsByClassName("main-heading");
```

Reply:
Did you mean <code>document.getElementsByClassName("main-heading")<strong>[0]</strong></code>?

Reply:
I did and I realize it's less good, so please ignore this comment. ^^'